### ISSUE TITLE
[Bug]: Cannot remove node (running MicroK8s) from Netdata Cloud after Helm reinstall

### ISSUE THREAD

**Issue by phuna** [netdata/netdata#13400]
### Bug description

I have a node running Ubuntu 22.04, MicroK8s 1.24.0 with Helm3 & NFS addons enabled. 
After I installed Netdata using Helm chart method & register it with Netdata Cloud, I can't remove it from Netdata Cloud. I tried to uninstall then reinstall it without registering to Netdata Cloud but my node still appears in Netdata Cloud.

### Expected behavior

My node should not be displayed in "All nodes" screen of Netdata Cloud after reinstall Netdata and not register with Netdata Cloud.

### Steps to reproduce

I follow instruction here: https://learn.netdata.cloud/docs/agent/packaging/installer/methods/kubernetes

1. `microk8s helm3 repo add netdata https://netdata.github.io/helmchart/`
2. `microk8s helm3 install netdata netdata/netdata`
3. Create override.yml file with content as shown in "Add nodes to All nodes" screen on app.netdata.cloud
4. `helm upgrade -f override.yml netdata netdata/netdata`
5. Wait a few seconds and verify that node metrics are shown in Netdata Cloud.
6. `helm3 uninstall netdata`
7. Run `kubectl get pv` to ensure all netdata related PVs are deleted
8. `helm3 install netdata netdata/netdata`
9. Check Netdata Cloud, expect that node's metrics are not displayed

### Installation method

helmchart (kubernetes)

### System info

```shell
/etc/lsb-release:DISTRIB_ID=Ubuntu
/etc/lsb-release:DISTRIB_RELEASE=22.04
/etc/lsb-release:DISTRIB_CODENAME=jammy
/etc/lsb-release:DISTRIB_DESCRIPTION="Ubuntu 22.04 LTS"
/etc/os-release:PRETTY_NAME="Ubuntu 22.04 LTS"
/etc/os-release:NAME="Ubuntu"
/etc/os-release:VERSION_ID="22.04"
/etc/os-release:VERSION="22.04 LTS (Jammy Jellyfish)"
/etc/os-release:VERSION_CODENAME=jammy
/etc/os-release:ID=ubuntu
/etc/os-release:ID_LIKE=debian
/etc/os-release:UBUNTU_CODENAME=jammy
```

### Netdata build info

```shell
Version: netdata v1.35.1
Configure options:  '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var' '--libexecdir=/usr/libexec' '--libdir=/usr/lib' '--with-zlib' '--with-math' '--with-user=netdata' '--without-bundled-protobuf' '--disable-dependency-tracking' '--with-bundled-libJudy' '--disable-ebpf' 'CFLAGS=-O2 -pipe' 'LDFLAGS='
Install type: oci
    Binary architecture: x86_64
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES
    ACLK Next Generation:       YES
    ACLK-NG New Cloud Protocol: YES
    ACLK Legacy:                NO
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         YES
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    NO
    EBPF:                    NO
    IPMI:                    YES
    NFACCT:                  NO
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 YES
    Prometheus Remote Write: YES
```

**Comment by ilyam8** [comment]
Hi, @phuna. I think you have claimed children nodes too, not only the parent, correct? If so 
- every child instance uses `/var/lib/netdata-k8s-child` [hostPath](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath) mount to store `/var/lib/netdata` (see [child.persistence](https://github.com/netdata/helmchart/blob/43f6f30a9708e435bfdb7548779fa47d97fadca4/charts/netdata/values.yaml#L262-L264))
- the dir which contains `cloud.d/`.
- every time you install Netdata the dir gets mounted to a new instance and the instance uses the data to connect to the Cloud via ACLK.

Using hostPath is the way to persist data (not only Cloud related but anything that uniquely identifies the instance - used in streaming) for [Daemonset](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/).

TL;DR 

- you need to ssh to the host.
- `rm /var/lib/netdata-k8s-child/var/lib/netdata/cloud.d/*`.
- install/reinstall Netdata.

**Comment by phuna** [comment]
@ilyam8 that works, thanks a lot for your prompt reply.
